modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -25,25 +25,38 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 For security, Red Hat recommends first setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
 
-. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. For example:
-.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
-+
+. Optional: To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section, then enter the rule information and click *Save*. You will need attributes for the user so that you can configure their access. To get user information in {ocp}, you can use one of the following methods:
```

Comment:
Maybe
```suggestion
. Optional: To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section, enter the rule information then click *Save*. You will need attributes for the user so that you can configure their access. To get user information in {ocp}, you can use one of the following methods:
```
Up to you though!